client/components/faq/docs/example.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { FAQ, FAQItem } from 'components/faq';
+
+export default React.createClass( {
+	displayName: 'FAQ',
+
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/design/faq">FAQ</a>
+				</h2>
+				<FAQ>
```

Comment:
@rralian asked for more `<FAQItems>`:
>  I think the example should include more items, so you it's more clear how the alignment works. With only three items you don't know for sure how the next set of items will align.

I also think it shows how the `<FAQ>` component behaves with more items in a better way if there are more items.